x/leveragelp/spec/incentivized_liquidation_system.md

```diff
@@ -0,0 +1,44 @@
+
+# Incentivized Liquidation System for Leverage Market
+
+## Diagram
+
+```plaintext
++------------+                  +------------+                  +-------------+                 
+|            |                  |            |                  |             |                
+| Depositors |                  | Borrowers  |                  | Liquidators |                
+|            |                  |            |                  |             |                  
++-----+------+                  +-----+------+                  +-----+-------+               
+      |                               |                               |                             
+      | Deposit stable coins          | Borrow stable coins           | Check positions                
+      | and earn interest             | to open leverage positions    | for liquidation or if certain price is met(stop loss)     
+      |                               |                               |                              
++-----v------+                  +-----v------+                  +-----v-------+                   
+|            |                  |            |                  |             |                 
+|  Stable    |<-----------------| Leverage   |<-----------------| Liquidation |
+|  Coin Pool |   Borrow coins   | Position   |   Monitor        |   Function  |   Monitor        
+|  Module    |                  | Module     |   positions      |             |   positions       
++------------+                  +------------+                  +-------------+                  
+      |                               |                               |                               
+      |                               |                               |                                
+      |  Interest                      | Position health calculated   | If health < threshold, or certain price is met(stop loss)     
+      |                               | as lpAmount / debt            | trigger liquidation           
```

Comment:
in the case of a SL it triggers a position closure not a liquidation

Reply:
updating